src/Abstractions/ServiceContext/IServiceContext.cs

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
This interface is practically unused, the only reason to keep it would be for the sake of completeness. Most of the context is exposed via IExecutionContext which pretends to be platform-agnostic, the 2 fields here are SF-specific and also not very useful. If really needed, they can be accessed via ```IAccessor<ServiceContext>```.

Reply:
If you want to keep the interface and both implementations, registration logic should be fixed.

I would either leave them where they are now or just delete.